bigipreport.ps1

```diff
@@ -1998,7 +2102,7 @@ do {
                         log $obj.severity ($job.name + ':' + $obj.message) $obj.datetime
                     } elseif ($obj["LoadBalancer"]) {
                         $Global:ReportObjects.add($obj.LoadBalancer.ip, $obj)
-                        Foreach ($thing in ("ASMPolicies", "Certificates", "DataGroups", "iRules", "Monitors", "Pools", "VirtualServers")) {
+                        Foreach ($thing in ("ASMPolicies", "Certificates", "DataGroups", "iRules", "Policies", "Monitors", "Pools", "VirtualServers")) {
```

Comment:
bigipreport.ps1:2105 foreach is sorted. "Policies" should be after "Monitors"

Reply:
updated